app/controllers/connections_controller.rb

```diff
@@ -2,15 +2,21 @@ class ConnectionsController < AuthenticationController
   before_action :ensure_profile_availability
 
   def new
+    @connection = Connection.new
     connections_profile = Profile.find(params[:profile_id])
+
     requester_name = current_user.profile.first_name + " " + current_user.profile.last_name
     requester_email = current_user.profile.contact_email
+    @connection.requester_id = current_user.id
+
     requested_name = connections_profile.first_name + ' ' + connections_profile.last_name
     requested_email = connections_profile.contact_email
+    @connection.requested_id = connections_profile.user.id
 
     if ConnectionMailer.connection_email(requester_name, requester_email, requested_name, requested_email).deliver_now
       ConnectionMailer.connection_email_copy(requester_name, requester_email, requested_name, requested_email).deliver_now
```

Comment:
Now that we are creating a Connections table and model, can we move the email sending/delivery to the model? maybe as an after_create hook? How do you think that will scale?